### PR TITLE
Gidk 977 stage

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@import "../../../../../assets/css/color.scss";
+.bs-table-grid {
+    th{
+        border: none !important;
+        border-bottom:1px solid $gainsboro !important;
+        padding: 13px 8px 14px 0 !important;
+        &:first-child{
+            padding: 0 !important;
+        }
+            i.glyphicon-search{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss:10]: Do run formatter

File: apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
-    margin-top: 15px;
-}
+@import "../../../../../assets/css/color.scss";
 .liabilities-assets {
     max-width: 760px;
     width: 100%;
 }
+.trial-balance{
+    span{
+        i.glyphicon-search{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss:8]: Do run formatter

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss]: done

**Pull request merged**